rbac/rbac/settings.py

```diff
@@ -213,10 +214,18 @@
     "ORDERING_PARAM": "order_by",
 }
 
-CW_AWS_ACCESS_KEY_ID = ENVIRONMENT.get_value("CW_AWS_ACCESS_KEY_ID", default=None)
-CW_AWS_SECRET_ACCESS_KEY = ENVIRONMENT.get_value("CW_AWS_SECRET_ACCESS_KEY", default=None)
-CW_AWS_REGION = ENVIRONMENT.get_value("CW_AWS_REGION", default="us-east-1")
-CW_LOG_GROUP = ENVIRONMENT.get_value("CW_LOG_GROUP", default="platform-dev")
+# CW settings
+if ENVIRONMENT.bool("CLOWDER_ENABLED", default=False):
```

Comment:
Just a note on the CW integration with RBAC. We currently have this disabled, by way of having null values for the keys in our environments, because we ran into issues where we weren't able to uses parallel processing, and the synchronous CW delivery was killing our pods. https://github.com/RedHatInsights/insights-rbac/pull/302#issuecomment-656615590

Reply:
We could put another environment variable on the template here to simply disable the logging altogether?

Reply:
I think that would be great! That would clear up some ambiguity around whether or not we'd got it disabled. 😄